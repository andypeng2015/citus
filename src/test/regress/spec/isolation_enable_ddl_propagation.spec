setup
{
	SELECT run_command_on_workers('ALTER SYSTEM SET citus.enable_ddl_propagation TO ON');
	SELECT run_command_on_workers('SELECT pg_reload_conf()');
}

session "s1"
step "s1-begin"
{
	BEGIN;
}

step "s1-commit"
{
	COMMIT;
}

permutation "s1-begin" "s1-commit"
