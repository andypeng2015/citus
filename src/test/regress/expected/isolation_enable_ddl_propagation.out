Parsed test spec with 1 sessions

starting permutation: s1-begin s1-commit
run_command_on_workers
---------------------------------------------------------------------
(localhost,57637,t,t)
(localhost,57638,t,t)
(2 rows)

step s1-begin:
 BEGIN;

step s1-commit:
 COMMIT;

